ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon__v4.13.yaml

```diff
@@ -0,0 +1,56 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  ocp-addons-operators-cli:
+    name: ocp-addons-operators-cli
+    namespace: ci
+    tag: latest
+  rhods-addon-tests:
+    name: ods-ci
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: smoke-rosa-rhods
+  cron: 0 12 * * 6
+  remote_api: true
+  steps:
+    cluster_profile: aws-managed-cspi-qe
+    env:
+      ADDON1_CONFIG: name=managed-odh;notification-email=interop-qe-ms@redhat.com
+      CLUSTER_NAME: ods-add-413
```

Comment:
please rename to `odh-add-413` or similar

Reply:
done